### ISSUE TITLE
Lighthouse error occurs when adding Web Stories shortcode in widget

### ISSUE THREAD

**Issue by LuckynaSan** [ampproject/amphtml#31110]
## What's the issue?

Embedding a story using Web Stories shortcode in a widget causes a lighthouse error: <frame> or <iframe> elements do not have a title.

Reporting this issue in the amphtml repo as this is using amp-story-player.

## How do we reproduce the issue?

1. Create Web Story. Publish. Copy shortcode for story post.
     Example: [web_stories_embed url="http://testsite.cmsdevrel.com/web-stories/901/" title="test" poster="http://testsite.cmsdevrel.com/wp-content/uploads/2020/08/image-524-1.png" width="360" height="600" align="none"]
2. Add "Text" widget. Paste shortcode for story post.
3. Run lighthouse report. See [report](https://jmp.sh/if4KhF8).

## What browsers are affected?

Not specific to browser or device type.

**Comment by swissspidy** [comment]
FWIW this uses `amp-story-player`, so it's actually an issue with that component and can be reproduced by inserting `<amp-story-player>` and running Lighthouse.

cc @Enriqe

**Comment by Enriqe** [comment]
@swissspidy how does the widget create the player? I'm asking because given the following configuration:
```
<amp-story-player>
  <a href="my-story.html">My Cool Title</a>
</amp-story-player>
```

The player will use "My Cool Title" provided between the `<a>` tags as title for the iframe. Or alternatively if the story is added via `add(storyObj)`, the `storyObj` passed should contain a title for it to be added to the iframe.

**Comment by swissspidy** [comment]
@Enriqe yeah that's the markup we print in the plugin.

I just tried reproducing this locally and was unable to do so. Perhaps this got fixed since initial reporting.

**Comment by Enriqe** [comment]
Ok, closing for now but feel free to reopen if it pops up again.
